apps/barnepensjon-ui/src/locales/en.ts

```diff
@@ -10,6 +10,10 @@ const common = {
     lastName: 'Family name',
     name: 'Name',
     fnrDnr: 'Norwegian national identity number or D number',
+    fnrIsUnknown: 'I do not know the national identy number. (optional)',
```

Comment:
Småpirk, men er det meningen at det skal være punktum i tekstene for barnepensjon og ikke oms ref `number. (optional)`

Reply:
Bare klipp og lim fra mural 🤷‍♂️ 
Jeg synes det ser bedre ut _uten_ punktum i disse tilfellene, men det er nå meg.

Reply:
Jeg er helt enig med deg, så kan gjerne fjerne punktum fra alle språk